src/com/spikes2212/Cameras/CamerasHandler.java

```diff
@@ -29,7 +36,7 @@ public synchronized void stop() {
 	}
 
 	public synchronized void start(int index) {
-		if (currentCamera != cameras.get(index)) {
+		if (currentIndex != index) {

```

Comment:
in conjunction with the way addCamera is written, this means you can't start a camera if it's the last one you added, which is problematic